fuel-vm/src/interpreter/blockchain/test.rs

```diff
@@ -6,10 +6,10 @@ mod scwq;
 mod srwq;
 mod swwq;
 
-fn mem(chains: &[&[u8]]) -> Vec<u8> {
+fn mem(chains: &[&[u8]]) -> Box<[u8; VM_MEMORY_SIZE]> {
```

Comment:
Hmm, we have `Box<[u8; VM_MEMORY_SIZE]>` in many places. Maybe we want to add an alias like `VMMemory` or `Memory` or `Buffer` or `VMBuffer`

Reply:
We can although I do think it's good seeing the size explicitly. It makes it clear how much memory you have when indexing etc.

Reply:
You are right=) I didn't know this

<img width="352" alt="image" src="https://user-images.githubusercontent.com/18346821/221281671-b8b62219-a0fb-4963-b894-0db1be27a904.png">

In most cases, this information still is not very actual, so I prefer an alias. But if it gives some benefits to you during development then it's up to you =)

Reply:
What about:
```rust
pub type Memory<const SIZE: usize> = Box<[u8; SIZE]>;

fn foo() -> Memory<VM_MEMORY_SIZE> {}
```
The we still see the size in the type but hide the other details?

Reply:
Works for me=) [nit] `VM_MEMORY_SIZE` seems soooooo long for me=D

Reply:
`MEM_SIZE`?